kafka-zookeeper/Dockerfile

```diff
@@ -1,42 +1,39 @@
-FROM alpine:3.12 AS kafka
+# This Dockerfile and supporting content derived originally from
+# https://github.com/openzipkin/zipkin/blob/master/docker/collector/kafka/
 
-ENV SCALA_VERSION 2.12
-ENV KAFKA_VERSION 2.3.0
+# Using the same image as we use in CircleCI to avoid transfer costs
+FROM cimg/openjdk:14.0.2 AS install
 
-WORKDIR /opt/kafka
-COPY install.sh /opt/kafka/install
-RUN /opt/kafka/install && rm /opt/kafka/install
+# Use latest stable release here. Scala 2.13+ supports JRE 14
+ENV KAFKA_VERSION=2.6.0 SCALA_VERSION=2.13
 
-COPY wait-for-zookeeper.sh /opt/kafka/bin
-COPY start.sh /opt/kafka/bin
+WORKDIR /install
+ADD docker/collector/kafka/install /tmp/install
+RUN /tmp/install && rm /tmp/install
 
-# Share the same base image to reduce layers
+# Share the same base image to reduce layers used in testing
 FROM hypertrace/java:11
-
 MAINTAINER Hypertrace "https://www.hypertrace.org/"
 
+# Add HEALTHCHECK and ENTRYPOINT scripts into the default search path
+COPY docker/collector/kafka/docker-bin/* /usr/local/bin/
```

Comment:
check source path. it should be `docker-bin/*`.

Reply:
indeed fixed and tested it locally.